voltaire_bundler/user_operation/user_operation_handler.py

```
@@ -1057,6 +1057,43 @@ async def get_transaction_by_hash(
+    try:
+        if selector == bytes.fromhex("08c379a0"):  # Error(string)
+            reason: str = decode(["string"], params)[0]
+            return reason
+        elif selector == bytes.fromhex("4e487b71"):  # Panic(uint256)
+            code: int = decode(["uint256"], params)[0]
+            return (
+                f"panic: {PANIC_REASONS.get(code, 'unknown panic code')} "
+                f"(0x{code:02x})"
+            )
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)